adbe/adb_enhanced.py

```diff
@@ -1658,21 +1660,20 @@ def alarm_manager(param):
 
                 action = value.split(":")[1]
 
-                print("Package name: %s" % package_name)
-                print("   Action: %s" % action)
-                print("   Running time: %s" % running_time)
-                print("   Number of device woke up: %s" % nb_woke_up)
-                print("   Number of alarms: %s" % nb_alarms)
-                print("   User ID: %s" % uid)
+                print("%sPackage name: %s" % (padding, package_name))
+                print("%s   Action: %s" % (padding * 2, action))
```

Comment:
Consider using "\t" instead of "   "?

Reply:
I tried "\t" but it produce a lot of spaces. I will replace it with "\t"